.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,38 @@
+# Pull Request
+
+## Summary
+
+Provide a concise and clear summary of your pull request.
+
+## Motivation
+
+Explain the motivation behind this pull request. What problem does it solve or feature does it introduce?
+
+## Changes Made
+
+Describe the changes made in this pull request. Be specific about code changes, new features, or bug fixes.
+
+## Related Issues/PRs
+
+Link to any related issues or pull requests, if applicable.
+
+## Screenshots (Before/After if applicable)
+
+Include screenshots or visual aids to demonstrate the changes visually. If your PR modifies the UI or behavior, include before and after descriptions or screenshots.
+
+### Desktop
+
+### Mobile
+
+### iPad
+
+## Checklist
+
+- [x] I have read the [Contributing Guidelines](../../CONTRIBUTING.md) and followed them.
+- [ ] I have tested my changes thoroughly.
+- [ ] My code follows the project's coding standards and style guide.
+- [ ] I have updated documentation (if applicable).
+
+## Any Other Context
```

Comment:
suggestion: I find this section not very useful. Personally, I'd expect the authors to add this directly in the "changes made" or as part of the PR comments. What do you think?